app/src/main/java/net/geekinpurple/www/appofourown/Work.java

```diff
@@ -142,8 +136,17 @@ public Work(Element work) {
             this.summary = "";
         }
         Log.d("testing", this.summary);
+
+        //Parse for work's Tags
+        Elements ships = work.getElementsByClass("relationships");
```

Comment:
Loop does nothing, and ships makes me think of starships or aircraft carriers not relationships. Slang can be problematic in code.